### DISCUSSION TITLE
入门小白求助，初始化 taro-ui 模板，npm install 不成功，依赖报错

### DISCUSSION THREAD

**Discussion by guoyp1985** [NervJS/taro#16276] [Q & A] [answered]
![报错](https://github.com/user-attachments/assets/aea5b28b-4624-4cb2-ace6-519569e13950)

**Comment by TheKonka** [comment, marked as answer]
换 yarn 或者 pnpm 试试， 说实话 taro-ui 很久不更新了，不推荐使用

**Reply by guoyp1985** [reply]
可是选别的redux模板也会类似的依赖包不匹配的情况

**Reply by TheKonka** [reply]
> 可是选别的redux模板也会类似的依赖包不匹配的情况

确实是会报错，貌似react-refresh 版本问题，可以直接改成版本为 0.14.2， 也可以先用yarn或者pnpm，或者npm i --force，这个问题暂时不影响使用，后面会修复

**Reply by guoyp1985** [reply]
执行npm i --force 报错更多，我试下选择别的模板

**Comment by honlyHuang** [comment]
用 NutUI 吧

**Reply by guoyp1985** [reply]
嗯嗯，朋友也推荐这个
